### DISCUSSION TITLE
What is the difference between bull and bullmq. They both mention Taskforce.sh

### DISCUSSION THREAD

**Discussion by asuraphel** [taskforcesh/bullmq#1522] [General]
They look like they are managed by the same company but under different Github org. What is their difference? Thanks

**Comment by manast** [comment]
BullMQ is the new codebase written in Typescript and the one that will receive newer features, while Bull is the old codebase where we only perform maintenance work.

**Reply by asuraphel** [reply]
Thanks. It'd would be nice if this was described in Bull's repo. Are they compatible though? If I switch to BullMQ will scheduled jobs run?

**Reply by manast** [reply]
Most likely yes, but since the codebases have diverged during a couple of years it is not guaranteed. The best is to start with new queues if you are migrating to BullMQ.

**Reply by asuraphel** [reply]
Okay. Thank you!

**Reply by v29neil** [reply]
@manast Is there any performance improvement in BullMQ , I was using Bull earlier , Have noticed some of the Jobs were getting missed in processing.  Can i expect BullMQ to cover this case?

**Reply by manast** [reply]
There are no such known issues in BullMQ, if you find anything you are welcome to post an issue.
